### PR TITLE
Rename istio-ingress-controller to istio-ingress

### PR DESCRIPTION
**Pull request by andraxylia**

### CODE REVIEW

File: kubernetes/istio-15.yaml [hunk 1]
```diff
@@ -232,17 +232,17 @@ spec:
 apiVersion: v1
 kind: Service
 metadata:
-  name: istio-ingress-controller
+  name: istio-ingress
   labels:
-    infra: istio-ingress-controller
+    infra: istio-ingress
```
Comment (ayj) [inline kubernetes/istio-15.yaml]: Does it make sense to unify the label key here with other Istio components, e.g. ""istio: ingress", "istio: manager", "istio: mixer"?

Comment (andraxylia) [inline kubernetes/istio-15.yaml]: Done for both ingress and egress.

File: kubernetes/istio-15.yaml [hunk 2]
```diff
@@ -83,7 +83,7 @@ data:
         params:
           quotas:
           - descriptor_name: RequestCount
-            max_amount: 5
+            max_amount: 5000
```
Comment (ayj) [inline kubernetes/istio-15.yaml]: Bad merge?

Comment (andraxylia) [inline kubernetes/istio-15.yaml]: Nope, this is a generated file. Somebody changed that value in mixer.yaml and did not run updateVersions.sh.

**Pull request merged**